tests/kafkatest/services/streams.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, test_context, kafka, command):
 
     def clean_node(self, node):
         if self.clean_node_enabled:
-            super.clean_node(self, node)
+            super(StreamsEosTestBaseService, self).clean_node(node)
```

Comment:
So, this isn't valid python3... It's also the only `super` invocation in kafkatest that's formatted this way. I'm not sure if it's valid python2.

But it only affects cleanup for the Eos test, and it just generates a warning instead of failing the test. So I can see how this could go unnoticed if the tests are usually run from a clean state instead of cleaning up after themselves.